docs/api/testing/how-to-test-the-prime-api.md

```diff
@@ -144,3 +144,6 @@ prime-api payment-requests POST data.json
 ```
 
 Note that this method for accessing the API is not actively being supported and may be less reliable than others (Postman or the Prime API Client CLI).
+
+## References
+* [Acceptance testing payment requests](/docs/backend/testing/acceptance-testing-payment-requests)
```

Comment:
If testing this link from the GH view file page. After clicking not he link you will have to append `.md` to the file for it to show, however in Docusaurus using `.md` will not work.

Reply:
Weird. I referenced our internal docs for linking. Which the official Docusaurus documentation says that file based paths work and will be cleaned up automatically. 

=> https://transcom.github.io/mymove-docs/docs/tools/docusaurus/docusaurus#linking-between-pages

Reply:
@rogeruiz it works in Docusaurus, it doesn't seem to work in GH (when I tested it).